### PR TITLE
Reuse one `Dockerfile` for all CentOS 7 builds

### PR DESCRIPTION
**Pull request by jakirkham**
* Synchronize content across the different CentOS 7 architecture builds
* Generalize `Dockerfile` to across architectures
* Configure CI to build with one `Dockerfile` for CentOS 7 builds

### CODE REVIEW

**Comment by jakirkham** [comment]
Thoughts @conda-forge/core? 🙂

File: linux-anvil-cos7/Dockerfile [hunk 1]
```diff
@@ -1,24 +1,36 @@
...
+ENV DISTRO_ARCH=${DISTRO_ARCH} \
+    DISTRO_NAME=${DISTRO_NAME} \
     DISTRO_VER=${DISTRO_VER}
 
 # Set an encoding to make things work smoothly.
 ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US.UTF-8
+
+# bust the docker cache so that we always rerun the installs below
+ADD https://loripsum.net/api /opt/docker/etc/gibberish
```
Comment (xhochy) [inline linux-anvil-cos7/Dockerfile:22]: Any reason you moved the cache busting before the qemu binaries?

Comment (jakirkham) [reply linux-anvil-cos7/Dockerfile:22]: The original idea of busting the cache was to make sure the images were rebuilt from scratch from the beginning (particularly on services like DockerHub, which always use a cache to build)

So had moved this before other steps

Though if there are other considerations, would be interested to learn more about them

**Review by isuruf — APPROVED** [review]

**Comment by jakirkham** [comment]
@xhochy did you have any more thoughts on this one? 🙂

**Review by xhochy — APPROVED** [review]

**Comment by jakirkham** [comment]
Thank you both! 🙏

**Pull request merged**